<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stationary7</class>
 <widget class="QWidget" name="Stationary7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 7 Stationary</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Profiles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Ultra Sun</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileTick">
        <property name="text">
         <string>Tick</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileTickValue">
        <property name="text">
         <string>383e329</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileShinyCharm">
        <property name="text">
         <string>Shiny Charm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileShinyCharmValue">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileOffset">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileOffsetValue">
        <property name="text">
         <string>17</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxStationaryRNGInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>RNG Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStationaryStartDate">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelStationaryEndDate">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelStationaryFrame">
        <property name="text">
         <string>Frame:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QDateTimeEdit" name="dateTimeEditStationaryEndDate">
        <property name="displayFormat">
         <string>M/d/yyyy H:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="TextBox" name="textBoxStationaryStartFrame">
        <property name="text">
         <string notr="true">478</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QDateTimeEdit" name="dateTimeEditStationaryStartDate">
        <property name="displayFormat">
         <string>M/d/yyyy H:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QPushButton" name="pushButtonStationarySearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="TextBox" name="textBoxStationaryEndFrame">
        <property name="text">
         <string notr="true">10000</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QPushButton" name="pushButtonStationaryCancel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxStationarySettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QLabel" name="labelStationarySynchNature">
          <property name="text">
           <string>Synch Nature</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBoxStationaryGenderRatio"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelStationaryGenderRatio">
          <property name="text">
           <string>Gender Ratio</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxStationarySynchNature"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxStationaryAlwaysSynch">
          <property name="text">
           <string>Always Synched</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBoxStationary3IVs">
          <property name="text">
           <string>3 IVs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBoxStationaryShinyLock">
          <property name="text">
           <string>Shiny Locked</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxStationaryAbilityLock">
        <property name="text">
         <string>Ability Locked</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxStationaryAbilityLock">
        <item>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">H</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBoxStationaryFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0" rowspan="5">
       <widget class="IVFilter" name="ivFilterStationary" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelStationaryNature">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CheckList" name="comboBoxStationaryNature"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonStationaryAnyNature">
        <property name="text">
         <string>Any</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelStationaryHiddenPower">
        <property name="text">
         <string>Hidden Power</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="CheckList" name="comboBoxStationaryHiddenPower"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonStationaryAnyHiddenPower">
        <property name="text">
         <string>Any</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelStationaryAbility">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBoxStationaryAbility">
        <item>
         <property name="text">
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">H</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelStationaryGender">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBoxStationaryGender">
        <item>
         <property name="text">
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">♂</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">♀</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelStationaryShiny">
        <property name="text">
         <string>Shiny</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="checkBoxStationaryShiny">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBarStationary">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="TableView" name="tableViewStationary">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>IVFilter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/IVFilter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/CheckList.hpp</header>
   <slots>
    <slot>resetChecks()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>dateTimeEditStationaryStartDate</tabstop>
  <tabstop>dateTimeEditStationaryEndDate</tabstop>
  <tabstop>textBoxStationaryStartFrame</tabstop>
  <tabstop>textBoxStationaryEndFrame</tabstop>
  <tabstop>pushButtonStationarySearch</tabstop>
  <tabstop>pushButtonStationaryCancel</tabstop>
  <tabstop>comboBoxStationarySynchNature</tabstop>
  <tabstop>comboBoxStationaryGenderRatio</tabstop>
  <tabstop>checkBoxStationaryAlwaysSynch</tabstop>
  <tabstop>checkBoxStationary3IVs</tabstop>
  <tabstop>checkBoxStationaryShinyLock</tabstop>
  <tabstop>checkBoxStationaryAbilityLock</tabstop>
  <tabstop>comboBoxStationaryAbilityLock</tabstop>
  <tabstop>comboBoxStationaryNature</tabstop>
  <tabstop>pushButtonStationaryAnyNature</tabstop>
  <tabstop>comboBoxStationaryHiddenPower</tabstop>
  <tabstop>pushButtonStationaryAnyHiddenPower</tabstop>
  <tabstop>comboBoxStationaryAbility</tabstop>
  <tabstop>comboBoxStationaryGender</tabstop>
  <tabstop>checkBoxStationaryShiny</tabstop>
  <tabstop>tableViewStationary</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonStationaryAnyNature</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxStationaryNature</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1111</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>1009</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStationaryAnyHiddenPower</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxStationaryHiddenPower</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1095</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
